### PR TITLE
Rename es-monitor-api to console-api

### PR DESCRIPTION
**Pull request by jsravn**
Preserve /service/es-monitor-api for backwards compatibility with
es-console-spa.

Also preserve the existing values.yml parameters, so we don't break
users that have these values encoded somewhere. That kind of change will
require a breaking release.

### CODE REVIEW

**Comment by jsravn** [comment]
Reviewers: need to be careful not to break backwards compatibility here, while trying to avoid too many lingering "es-monitor-api" words. I think I've covered the edge cases.

File: enterprise-suite/gotests/testenv/testenv.go [hunk 1]
```diff
@@ -141,7 +141,7 @@ func InitEnv() {
 	}
 
 	PrometheusAddr = fmt.Sprintf("%v/service/prometheus", ConsoleAddr)
-	MonitorAPIAddr = fmt.Sprintf("%v/service/es-monitor-api", ConsoleAddr)
```
Comment (marcoderama) [inline enterprise-suite/gotests/testenv/testenv.go]: Might there be value in continuing to test the old endpoint as well for now, especially given these changes?

File: enterprise-suite/tests/prometheus_common [hunk 2]
```diff
@@ -20,7 +20,7 @@ pods_up() {
 
 ES_CONSOLE=$( busy_wait nodeport )
 PROM="$ES_CONSOLE/service/prometheus"
-ES_MONITOR_API="$ES_CONSOLE/service/es-monitor-api"
+ES_MONITOR_API="$ES_CONSOLE/service/console-api"
```
Comment (marcoderama) [inline enterprise-suite/tests/prometheus_common]: You didn't dare change the env var name I see.  :-)

File: enterprise-suite/tests/smoke_nginx [hunk 3]
```diff
@@ -13,7 +13,7 @@ BASEPATH="${CONSOLE}/fee/fie/fou/fum"
 test_es_console_responding "$BASEPATH/"
 test_grafana_responding "$BASEPATH/service/grafana/"
 test_prom_responding "$BASEPATH/service/prometheus/"
-test_es_monitor_API_responding "$BASEPATH/service/es-monitor-api/"
+test_es_monitor_API_responding "$BASEPATH/service/console-api/"
```
Comment (marcoderama) [inline enterprise-suite/tests/smoke_nginx]: This should call `test_console_API_responding` now I think.

File: enterprise-suite/values.yaml [hunk 4]
```diff
@@ -3,8 +3,8 @@
 #
 # lightbend-docker-commercial-registry.bintray.io/enterprise-suite/es-console
 esConsoleVersion: v1.0.1
-# lightbend-docker-commercial-registry.bintray.io/enterprise-suite/es-monitor-api
-esMonitorVersion: v1.0.9
+# lightbend-docker-commercial-registry.bintray.io/enterprise-suite/console-api
```
Comment (marcoderama) [inline enterprise-suite/values.yaml]: Should the version bump go in a separate PR or is that just a function of the name change?

File: enterprise-suite/values.yaml [hunk 5]
```diff
@@ -27,8 +27,8 @@ busyboxVersion: "1.30"
 #
 # es-console image
 esConsoleImage: '{{ .Values.imageCredentials.registry }}/enterprise-suite/es-console'
-# es-monitor-api image
-esMonitorImage: '{{ .Values.imageCredentials.registry }}/enterprise-suite/es-monitor-api'
+# console-api image
+esMonitorImage: '{{ .Values.imageCredentials.registry }}/enterprise-suite/console-api'
```
Comment (marcoderama) [inline enterprise-suite/values.yaml]: Do we want to deprecate this and use `consoleImage` here?  (And ditto for `esMonitorVersion` -> `consoleVersion` above?)

File: enterprise-suite/gotests/testenv/testenv.go (hunk 1 above)
Comment (jsravn) [reply enterprise-suite/gotests/testenv/testenv.go]: Sure - I will add a basic backwards compatibility check.

File: enterprise-suite/values.yaml (hunk 4 above)
Comment (jsravn) [reply enterprise-suite/values.yaml]: It's related to the name change, since that is the version that changes to console-api.

File: enterprise-suite/values.yaml (hunk 5 above)
Comment (jsravn) [reply enterprise-suite/values.yaml]: I think that should be a separate issue - to remove all the `es` prefixes.
